Cargo.toml

```diff
@@ -22,9 +22,9 @@ hyper = "^0.10"
 hyper-openssl = { version = "^0.2", optional = true }
 log = "^0.3"
 maplit = "^0.1"
```

Comment:
I think `maplit` was only used in places that have now been replaced with the `json!` macro from `serde_json`, so we can probably get rid of this now.